Applications/utils/jni/meson.build

```diff
@@ -0,0 +1,24 @@
+utils_inc = include_directories('./includes')
```

Comment:
Android.mk is missing for this? I mean how does Android.mk of application gets the libnntrainer-util.so from?

Reply:
fixed. thanks.